REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.3
+julia 0.4
```

Comment:
Yes, but now that master is on 0.5 I have been putting less effort into 0.3 compatibility